src/components/Section.tsx

```diff
@@ -18,7 +18,7 @@ interface SectionProps {
 }
 
 function Section(props: SectionProps): React.Component {
-  props.data.forEach(data => data.image = `/assets/${data.image}`);
+  props.data.forEach(data => data.image.indexOf('/assets/') == -1 ? data.image = `/assets/${data.image}` : data.image);
```

Comment:
why is this check necessary?

Reply:
because i found out that each rerender of this component was prepending `/assets/` and so the images would break after the first rerender lol